modules/widgets/top-posts.php

```diff
@@ -124,11 +133,18 @@ function form( $instance ) {
 				<li><label><input id="<?php echo $this->get_field_id( 'display' ); ?>-list" name="<?php echo $this->get_field_name( 'display' ); ?>" type="radio" value="list" <?php checked( 'list', $display ); ?> /> <?php esc_html_e( 'Image List', 'jetpack' ); ?></label></li>
 				<li><label><input id="<?php echo $this->get_field_id( 'display' ); ?>-grid" name="<?php echo $this->get_field_name( 'display' ); ?>" type="radio" value="grid" <?php checked( 'grid', $display ); ?> /> <?php esc_html_e( 'Image Grid', 'jetpack' ); ?></label></li>
 			</ul>
-		</p>
+		</p><?php
 
-		<p><?php esc_html_e( 'Top Posts &amp; Pages by views are calculated from 24-48 hours of stats. They take a while to change.', 'jetpack' ); ?></p>
+		do_action( 'jetpack_widget_top_posts_after_fields', array( $instance, $this ) );
```

Comment:
Sorry, I was thinking the documentation was for the add_action, not do_action hence I missed this. This has been added now.

Should I perhaps remove the documentation that I added for the add_action call?